src/test/training/supportbank/BankTest.java

```diff
@@ -0,0 +1,65 @@
+package training.supportbank;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import training.supportbank.Account;
+import training.supportbank.Bank;
+import training.supportbank.Transaction;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class BankTest {
+
+    Bank bank;
+
+    @BeforeEach
+    public void createBank() {
+        bank = new Bank();
+    }
+
+    @Test
+    public void addMulitpleAccountsDoesNotAddAccountIfNameAlreadyExists() {
+        Set<String> accounts = Sets.newHashSet("accountName", "anotherAccount");
+        bank.addMultipleAccounts(accounts);
+        Transaction transaction = new Transaction("01/01/2014", "accountName", "anotherAccount", "narrative", "10");

```

Comment:
A lot of the tests set up `Transaction` objects which are pretty arbitrary. Java doesn't have a good equivalent of FactoryBot, but it's sometimes worth creating your own [builder](https://dzone.com/articles/design-patterns-the-builder-pattern) class for building data. It can supply some sensible (possibly random) defaults so each test only has to specify the things it needs to check.